src/components/Signin.jsx

```diff
@@ -0,0 +1,265 @@
+                                width="30%"
+                              >
+                                SignIn
+                              </Button>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-danger rounded-pill shadow"
+                                width="60%"
+                                marginLeft="5px"
+                              >
+                                Forgot your password?
+                              </Button>
+                            </div>
+                            <div className="form-group text-center">
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignUp
+                              </Button>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <Seperator >
+                                <h2><span>OR</span></h2>
+                              </Seperator>
+                              <DisplayForLinks >
+                                <SocialLinks >
+                                  <li>
+                                    <SocialMediaLink  color ="white" bgColor="#3B5998" href="#" className="btn">
+                                      <i className="fa fa-facebook fa-fw"></i> Signin with Facebook
+                                    </SocialMediaLink>
+                                  </li>
+                                  <li >
+                                    <SocialMediaLink bgColor="black" color="white" hoverBgColor="darkgrey" href="#" className=" btn">
+                                      <i className="fa fa-github "></i> Signin with Github
+                                    </SocialMediaLink>
+                                  </li>
+                                  <li >
+                                    <SocialMediaLink color="blue" bgColor="lightblue" href="#" className=" btn">
+                                      <LinkedinIcon className="fa fa-linkedin-square " ></LinkedinIcon><LinkedinLinkContent >Signin with Linkedin</LinkedinLinkContent>
+                                    </SocialMediaLink >
+                                  </li>
+                                  <li >
+                                    <SocialMediaLink color="black" bgColor="white" border="1px solid gray" hoverBgColor="darkgray" href="#" className=" btn">
+                                      <i className="fa fa-google "></i> Signin with Google
```

Comment:
Use google+ icon. check for google authentication logo, apply that background color here. Use online color picker to choose color from image.

Reply:
i didnt understand can u message me in slack about it?, are you telling to choose a svg image for google?

Reply:
No, it should look like this. https://www.google.com/search?q=signin+with&tbm=isch&ved=2ahUKEwjNjs2Oh8HrAhXrhEsFHV6WB9QQ2-cCegQIABAA&oq=signin+with&gs_lcp=CgNpbWcQAzICCAAyAggAMgIIADICCAAyAggAMgIIADIGCAAQCBAeMgYIABAIEB4yBggAEAoQGDIGCAAQChAYOgcIIxDqAhAnOgQIIxAnOgUIABCxAzoICAAQsQMQgwE6BAgAEEM6BggAEAUQHlCrW1iIdmCRe2gBcAB4AIABmwGIAbcKkgEEMC4xMZgBAKABAaoBC2d3cy13aXotaW1nsAEKwAEB&sclient=img&ei=IKBKX83-C-uJrtoP3qyeoA0&bih=730&biw=1525&client=firefox-b-d#imgrc=44pUNu0Gc4ooMM